plugins/ornament-icons.js

```diff
@@ -0,0 +1,107 @@
+// @author         johtata
```

Comment:
Make sure these images are free to use. 
( just a warning, I didn't reviewed these )

Reply:
redid every image that _we_ "deliver"
while I also redid TOASTY, I am not sure about the original one.